app/services/active_storage/service/sdr_service.rb

```diff
@@ -15,6 +15,11 @@ def download(key, &block)
         end
       end
 
+      def delete(key)
+        Honeybadger.context(key)
+        Honeybadger.notify('SdrService#delete called unexpectedly - possible attempt to purge work in preservation')
```

Comment:
can this include the key in the message?  What is the key, a druid?  Or, if it is a hash, can the druid be in the message?

Reply:
I think in the context might be preferable, that way these errors stack on one another in the UI